lighthouse-cli/cli-flags.js

```diff
@@ -137,7 +137,6 @@ function getFlags(manualArgv) {
       // default values
       .default('chrome-flags', '')
       .default('output', ['html'])
-      .default('emulated-form-factor', 'mobile')
```

Comment:
This was an erroneous double default that makes it impossible to run `lr-desktop-config` from the CLI

Reply:
was the cli flag winning over the config setting? Yeah, that seems bad...

Reply:
does `output` have the same problem? I don't see any other of these defaults in `SharedFlagsSettings`